app.models.js

```diff
@@ -50,11 +50,18 @@ exports.findCommentsByArticleId = (articleIdNum) => {
     if(comments.rows.length === 0) {
       return Promise.reject({
         status: 404,
-        msg: "Sorry, I couldn't find that!",
+        msg: "Sorry, I couldn't find that article!",
       })
     }
     else {
     return comments.rows
     }
   })
+}
+
+exports.postCommentOnArticle = (username, body, article_id) => {
+  return db.query(`INSERT INTO comments(author, body, article_id) VALUES ($1, $2, $3) RETURNING *`, [username, body, article_id])
+  .then(({ rows }) => {
+    return rows
```

Comment:
Instead of sending the array back here, I would recommend making sure this is just the single posted comment object being sent back instead